store/src/java/com/zimbra/cs/service/mail/ParseMimeMessage.java

```diff
@@ -943,6 +944,18 @@ private static void attachPart(MimeMultipart mmp, ItemId iid, String part, Strin
         attachUpload(mmp, up, contentID, ctxt, null, (contentDesc == null || contentDesc.length == 0) ? null : contentDesc[0], contentDisposition);
     }
 
+    private static String getDisposition(String contentDisposition) {
+        // Content-Disposition takes one of two values, `inline' and `attachment'
+        if (contentDisposition == null) {
+            return null;
+        }
+        if (contentDisposition.toLowerCase().startsWith("i")) {
```

Comment:
I can see that value of contentDisposition will always be in lowercase as its set as Part.INLINE or Part.ATTACHEMENT as well as if this will always have only 2 values then we can remove 2nd else if statement can do in a single if or if -else statement.

Reply:
Thanks, updated.